kotlin-csstype/src/main/generated/csstype/StandardProperties.kt

```diff
@@ -0,0 +1,5 @@
+// Automatically generated - do not modify!
+
+package csstype
+
+external interface StandardProperties : StandardLonghandProperties
```

Comment:
Why are you skipped `StandardShorthandProperties`?

Reply:
Longhand properties first